### PR TITLE
infra(unicorn): use hashbang for scripts

### PR DESCRIPTION
**Pull request by ST-DDT**
Ref: #2439

- #2439

---

Enables the [`unicorn/no-process-exit`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-process-exit.md) lint rule.

### CODE REVIEW

File: scripts/generateLocales.ts [hunk 1]
```diff
@@ -366,6 +366,8 @@ async function main(): Promise<void> {
 }
 
 main().catch((e) => {
+  // Workaround until top level await is available
   console.error(e);
+  // eslint-disable-next-line unicorn/no-process-exit
```
Comment (xDivisionByZerox) [inline scripts/generateLocales.ts]: We could simply add a hashbang to the start of the file to satisfy the linter.

Comment (ST-DDT) [reply scripts/generateLocales.ts]: Done

**Review by xDivisionByZerox — COMMENTED** [review]
Just so I can understand better: You did put the hashbang in all files which are entry files for a script, correct?

**Comment by ST-DDT** [comment]
> Just so I can understand better: You did put the hashbang in all files which are entry files for a script, correct?

Yes.

**Review by xDivisionByZerox — APPROVED** [review]

**Comment by matthewmayer** [comment]
PR title should be updated to reflect adding hashbangs

**Comment by ST-DDT** [comment]
Like this?

**Review by matthewmayer — APPROVED** [review]

**Pull request merged**